third_party/rules_pycross/pycross/private/wheel_library.bzl

```diff
@@ -119,6 +129,34 @@ and py_test targets must specify either `legacy_create_init=False` or the global
 This option is required to support some packages which cannot handle the conversion to pkg-util style.
             """,
         ),
+        "patch_args": attr.string_list(
+            default = ["-p0"],
+            doc =
+                "The arguments given to the patch tool. Defaults to -p0, " +
+                "however -p1 will usually be needed for patches generated by " +
+                "git. If multiple -p arguments are specified, the last one will take effect.",
+        ),
+        "patch_dir": attr.string(
+            default = "",
+            doc = "The directory from which to invoke the patch_tool.",
+        ),
```

Comment:
Since the user is supplying patches and they can supply `patch_args`, is this `patch_dir` necessary?

Reply:
It is something that I found myself wanting several times when using `http_archive`'s patching functionality. I added it here because I figured it may come up again or someone else might want to use it.

Reply:
I am a fan of extending things when needed, so if we don't have a usecase yet, we should not add it as it increases the API surface we need to maintain. What is more, `http_archive` patching is very limited, so the experience from there might not exactly transfer.

Reply:
Removed.